src/libs/ontouml2db/IOntoUML2DBOptions.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ *
+ * Author: Gustavo Ludovico Guidoni
+ */
+
+import { StrategyType } from './strategies/StrategyType';
+import { DBMSType } from './file_generation/DMBSType';
+
+export interface IOntoUML2DBOptions {
```

Comment:
Interfaces are not classes (at least they were not in their design). Distinguishing them from other structures by adding the letter "I" allows you to easily identify what the structure stores. Since interfaces are not classes and removing the letter "I" becomes necessary to group the classes in a particular folder, such as constants.